src/components/TodoForm.vue

```diff
@@ -52,6 +61,7 @@ const newTodo = ref("");
 const popup = ref(false);
 const selectedTaskIndex = ref();
 const checkedTodos = ref<Todotype[]>([]);
+const handleSearch = ref("");
```

Comment:
Please use single quote instead of double. You can set this as config in you editor.